accounts/config.go

```diff
@@ -0,0 +1,19 @@
+package accounts
+
+import (
+	"crypto/ecdsa"
+)
+
+type EthConfig struct {
+	Passphrase string `required:"false" default:"" yaml:"pass_phrase"`
+	Keystore   string `required:"false" default:"" yaml:"key_store"`
+}
+
+func (c *EthConfig) LoadKey() (*ecdsa.PrivateKey, error) {
+	key, err := LoadKeys(c.Keystore, c.Passphrase)
```

Comment:
Just a thought - I think we need to make some config validation and postprocessing, where we can load keys for example and make some usefull work (not now though)